src/worker/start.rs

```diff
@@ -467,9 +467,9 @@ fn gather_configuration(opts: WorkerStartOpts) -> anyhow::Result<WorkerConfigura
         work_dir,
         log_dir,
         heartbeat_interval: opts.heartbeat.into_duration(),
+        hw_state_poll_interval: Some(Duration::from_millis(1000)),
```

Comment:
Could you make this configurable from the command line (in another PR?). Just so that we don't forget about it.

Reply:
I've taken a note, I'll follow up with a PR after this one